src/gui/advancedsettings.cpp

```diff
@@ -349,42 +353,75 @@ void AdvancedSettings::loadAdvancedSettings()
     spinBoxCache.setValue(session->diskCacheSize());
     updateCacheSpinSuffix(spinBoxCache.value());
     addRow(DISK_CACHE, tr("Disk cache"), &spinBoxCache);
+    item(DISK_CACHE, PROPERTY)->setToolTip(QString("<body><p>%1</p></body>")
+            .arg(tr("The disk write and read cache. If this is set to -1, the cache size is automatically "
+                    "set to the amount of physical RAM available in the machine divided by 8. "
+                    "If the amount of physical RAM cannot be determined, it's set to 16 MiB")));
     // Disk cache expiry
     spinBoxCacheTTL.setMinimum(15);
     spinBoxCacheTTL.setMaximum(600);
     spinBoxCacheTTL.setValue(session->diskCacheTTL());
     spinBoxCacheTTL.setSuffix(tr(" s", " seconds"));
     addRow(DISK_CACHE_TTL, tr("Disk cache expiry interval"), &spinBoxCacheTTL);
+    item(DISK_CACHE_TTL, PROPERTY)->setToolTip(QString("<body><p>%1</p></body>")
+            .arg(tr("The number of seconds from the last cached write to a piece in the write cache, "
+                    "to when it's forcefully flushed to disk")));
     // Enable OS cache
     checkBoxOsCache.setChecked(session->useOSCache());
     addRow(OS_CACHE, tr("Enable OS cache"), &checkBoxOsCache);
     // Guided read cache
     checkBoxGuidedReadCache.setChecked(session->isGuidedReadCacheEnabled());
     addRow(GUIDED_READ_CACHE, tr("Guided read cache"), &checkBoxGuidedReadCache);
+    item(GUIDED_READ_CACHE, PROPERTY)->setToolTip(QString("<body><p>%1</p></body>")
+            .arg(tr("Enables the disk cache to adjust the size of a cache line generated by peers "
```

Comment:
"Enables" seem strange to me here, maybe "Allow"? and maybe put "generated by peers" between commas?

Reply:
@thalieht:  Those changes would certainly improve clarity here.